website/blog/2023-05-04-i-gambled-against-react-and-lost/item.md

```diff
@@ -0,0 +1,167 @@
+---
+title: I gambled against React and lost (and I don’t regret a thing)
+slug: 2023-05-04-i-gambled-against-react-and-lost
+authors:
+    - name: Jens Langhammer
+      title: CTO at Authentik Security Inc
+      url: https://github.com/BeryJu
+      image_url: https://github.com/BeryJu.png
+tags:
+    - blog
+    - react
+    - lit
+    - frontend
+    - web framework
+    - api
+hide_table_of_contents: false
+---
+
+Back in 2018, I made a fateful decision: I chose to rebuild authentik using [Lit](https://lit.dev/) and not React.
+
+We like to think that technical decisions are primarily, well, technical, but some of the biggest consequences of these decisions come from how a technology is adopted and used – not the technology itself.
+
+So it was with React.
+
+In this post, I’ll explain why I made this decision, how it did and didn’t pay off, and why, ultimately, I don’t regret it. The point isn’t to sway you toward or away from React or to make an argument about web frameworks in general, but to encourage a discussion about the choices early-stage startups have to make.
+
+<!--truncate-->
+
+And when I say startups here, I’m not talking about the scale-ups still calling themselves startups or the startup darlings with more dollars of venture capital than they can burn through, I’m talking about people like me: developers working full-time on trying to build a company out of a popular project.
+
+## React vs. Lit: A brief history and a one-sided victory
+
+React was first released in 2013 and when I was building the initial open source project that would eventually become Authentik Security, React was a known option but not, as it arguably is now, a default.
+
+In 2015, Netflix and Airbnb started using React. Reactive Native for iOS came out that same year and later, React Native for Android. By 2018, the year I made the decision, 28.3% of developers had started using React.
+
```

Comment:
I'd suggest you add a caption here, to clearly label the chart as showing 2018 data. We wouldn't want people to read to quickly and miss that fact and get all twisted. Ditto on chart from 2022.